_posts/2023-12-24-update.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Week Ending December 24, 2023
+date: 2023-01-26 22:00:00 -0000
+slug: 2023-01-24-update
+---
+
+## Developer News
+
+SIG Release [updated](https://groups.google.com/a/kubernetes.io/g/dev/c/JfZQhip-R7U) it's roadmap for 2024 onwards. If you want to contribute, just reach out to SIG Release using the K8s Slack (#sig-release) or comment on any of the open GitHub issues.
+
+## Release Schedule
+
+**Next Deadline: 1.30 Cycle Begins, January 2024**
+
+We are in the holiday period between releases; dates for 1.30 have not been published yet. If you want to become a part of the [Release team](https://github.com/kubernetes/sig-release/issues/2377), apply soon!
+
+## Featured PRs
+
+
+## KEP of the Week
+
+### [KEP 4020: Unknown Version Interoperability Proxy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/4020-unknown-version-interoperability-proxy)
+
+This KEP proposes to solve an issue that arises when multiple apiservers with different versions are running in a cluster. All the different apiservers might not be able to server all the resources in all the different versions. This can happen when a cluster updgrade/downgrade is going or when a runtime-config changes. This proposed filter will proxy clients to an apiserver that is capable of handling the request from the client. The KEP proposes to use the existing `StorageVersion` API to figure out which group, version and resources an apiserver can serve. If the apiserver doesn't have the requested group/version/resource locally, the request will be proxied to one of the apiservers listed in the `ServerStorageVersion` object. 
```

Comment:
```suggestion
This KEP proposes to solve an issue that arises when multiple apiservers with different versions are running in a cluster. All the different apiservers might not be able to server all the resources in all the different versions. This can happen when a cluster upgrade/downgrade is going or when a runtime-config changes. This proposed filter will proxy clients to an apiserver that is capable of handling the request from the client. The KEP proposes to use the existing `StorageVersion` API to figure out which group, version and resources an apiserver can serve. If the apiserver doesn't have the requested group/version/resource locally, the request will be proxied to one of the apiservers listed in the `ServerStorageVersion` object. 
```